indico/htdocs/js/indico/jquery/ajaxdialog.js

```diff
@@ -308,6 +308,11 @@
                     dataType: 'json',
                     data: options.getExtraData.call(this, options.trigger),
                     beforeSubmit: function() {
+                        var evt = $.Event('ajaxDialog:validateBeforeSubmit');
```

Comment:
I guess you needed to use a new event to prevent the ajaxDialog's own beforeSubmit handler from showing the progress dialog?